src/ctia/http/routes/common.clj

```diff
@@ -50,6 +53,65 @@
    (s/optional-key :search_after) (describe [s/Str] "Pagination stateless cursor")
    (s/optional-key :limit) (describe Long "Pagination Limit")})
 
+(def default-ignored-search-fields
+  #{:language
+    :revision
+    :schema_version
+    :source
+    :source_uri
+    :timestamp
+    :tlp
+    :type})
+
+(def default-whitelisted-search-fields
+  #{:id
+    :title
+    :short_description
+    :description
```

Comment:
some entities are not [describable](https://github.com/threatgrid/ctim/blob/d9462cb83f656f04fc9a764033a3861b4e7dfb94/src/ctim/schemas/common.cljc#L234) like [AssetMapping](https://github.com/threatgrid/ctim/blob/d9462cb83f656f04fc9a764033a3861b4e7dfb94/src/ctim/schemas/asset_mapping.cljc#L23)

Reply:
Ah, you're right. I'm gonna try something different. I'll try to get the fields not from the `fields` (because it doesn't actually include all the fields entity has), I'll try to grab it directly from the Entity schema.

Reply:
@ereteog I shuffled a few things, if you can, please take another gander. 

I am now getting the fields from the entity schema, and it works now this way:

- You provide the schema and optionally fields to ignore
-  There's `default-ignored-search-fields`, for fields that always will be ignored. It was tempting to make it possible to explicitly allow  some fields (even if they are in that list), but this is not going to be used right now; If ever needed, the function can be extended later. 
- And we don't need `default-whitelisted-search-fields` anymore.

____

Something new I've discovered that I somehow missed previously, pls. take a look: https://github.com/threatgrid/ctia/pull/1127/files#diff-7d7da115f9a07f82eb46cabf135808b2835014f44605f098f3f8b152cc5d4f82R53

If you agree, I will incorporate that feature separately.

____

And then I will update the QA instructions.

Reply:
If we do it separately we have a risk of regression.